src/test/java/io/confluent/idesidecar/restapi/testutil/FakeSideCarProfile.java

```diff
@@ -0,0 +1,22 @@
+package io.confluent.idesidecar.restapi.testutil;
+
+import io.quarkus.test.junit.QuarkusTestProfile;
+import java.util.Map;
+
+public class FakeSideCarProfile implements QuarkusTestProfile {
+
+  /**
+   * Disable the access token filter to protest unit tests.
+   */
+  @Override
+  public Map<String, String> getConfigOverrides() {
```

Comment:
These values need to be updated to reflect real values, the "override" values I showcased in my screenshot was just for demonstration purposes.